test/image/mocks/pie_separators.json

```diff
@@ -0,0 +1,19 @@
+{
+  "data": [
+    {
+      "values": [
+        1234567.90,
+        321321,
+        242132.153243,
+        4231.21,
+        5111
+      ],
+      "type": "pie"
+    }
+  ],
+  "layout": {
+    "height": 300,
+    "width": 400,
+    "separators": "∆|"
```

Comment:
Hmm, I don't think `separators` need an image test of their own. @mdtusz can you add `"separators"` to an existing pie image mock instead?